CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+For general contributing guidelines, see the [Contributing Guide on DoneJS.com][].
+
+## Developing Locally
+
+### Setup
+
+Fork and clone this repository. See [Fork A Repo][] and [Cloning a repository][] for help.
+
+Install the packages defined in [`package.json`][]:
+
+```
+npm install
+```
+
+### Build
+
+There is no build step for this repo.
+
+### Test
+
+Tests are located in `test.js`, at the root of this repo.
+
+To run the tests, execute the `test` script defined in [`package.json`][]:
+
+```
+npm test
+```
+
+This will generate a directory and file at `temp/index.html` containing a source-code snippet in HTML:
+
+```html
+<pre><code>var str ='hello world';</code></pre>
+```
+
+The tests will make sure that the source-code snippet is correctly prettified by the plugin.
+
+You can also open the generated file in your web browser to see the result using `http-server`:
+
+```
+npm install -g http-server
+npm test && http-server
+```
+
+Now visit <http://127.0.0.1:8080/temp/index.html>
+
+## Getting Help
+
+[Our forums](http://forums.donejs.com) and [Gitter chat](https://gitter.im/donejs/donejs) are the best places to ask questions.
+
+:heart: __THANK YOU!__ :heart:
```

Comment:
Oh I misread the first comment here, and missed Justin's. So remove this section?